backend/FwLite/FwLiteShared/Projects/CombinedProjectsService.cs

```diff
@@ -120,4 +120,10 @@ public async Task CreateProject(string name)
     {
         await crdtProjectsService.CreateExampleProject(name);
     }
+
+    [JSInvokable]
+    public async Task DeleteProject(string name)
```

Comment:
Is there any risk of this being invoked by non-admins / non-devs?

Reply:
No, I don't think so. And it's not a security concern.
I expect we'll expose this for all users eventually.